pipelines/tools-staging-prod-infra.yaml

```diff
@@ -153,17 +153,30 @@ resource_types:
     repository: govsvc/terraform-resource
     tag: 0.13.0-beta.2
 
-
+- name: github
+  type: docker-image
+  source:
+    repository: "govsvc/concourse-github-resource"
+    tag: "0.0.1551114195"
 
 
 
 resources:
 
 - name: gsp-teams
-  type: git
+  type: github
   source:
     uri: https://github.com/alphagov/gsp-teams.git
+    organization: alphagov
+    repository: gsp-teams
+    github_api_token: ""
```

Comment:
github token will need to be set to a variable like `((github-api-key))` ... you can add a secret by running the gsp "info" pipeline in the deployer concourse and following the instructions in the task output